src/lib.rs

```diff
@@ -270,7 +115,7 @@ macro_rules! create_derive(
     ($feature:literal, $mod_:ident, $trait_:ident, $fn_name: ident $(,$attribute:ident)* $(,)?) => {
         #[cfg(feature = $feature)]
         #[proc_macro_derive($trait_, attributes($($attribute),*))]
-        #[doc(hidden)]
+        #[doc = include_str!(concat!("../doc/", $feature, ".md"))]
```

Comment:
I think this is as alright solution for now, but I'm not a fan of the duplication this causes. The `Add` doc is essentially copy pasted for every `Add`-like derive:
![image](https://user-images.githubusercontent.com/1162278/196009382-f9e463c3-d57f-4163-a7b0-9dcddfb7fd03.png)

I'm not sure how to make this better without making it worse in other ways. So I'm fine with this for now.